### PR TITLE
#1332

### PR DESCRIPTION
**Pull request by jibcore**
Done.

### CODE REVIEW

File: ShareX.HelpersLib/Colors/CIELab.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+{
+    public struct CIELab
+    {
+        public double L { get; set; }
+
+        public double A { get; set; }
+
+        public double B { get; set; }
+
+        public CIELab(double l, double a, double b) : this()
```
Comment (Jaex) [inline ShareX.HelpersLib/Colors/CIELab.cs]: What `this()` do here?

File: ShareX.HelpersLib/Colors/RGBA.cs [hunk 2]
```diff
@@ -114,6 +114,11 @@ public RGBA(Color color)
             return color.ToCMYK();
         }
 
+        public static implicit operator CIELab(RGBA color)
+        {
+            return color.ToCMYK();
```
Comment (Jaex) [inline ShareX.HelpersLib/Colors/RGBA.cs]: Why `ToCMYK()` here?

File: ShareX.ScreenCaptureLib/Forms/RectangleRegion.cs [hunk 3]
```diff
@@ -507,7 +507,7 @@ private string GetInfoText()
                         Replace("$y", CurrentPosition.Y.ToString(), StringComparison.InvariantCultureIgnoreCase);
                 }
 
-                return string.Format(Resources.RectangleRegion_GetColorPickerText, color.R, color.G, color.B, ColorHelpers.ColorToHex(color), CurrentPosition.X, CurrentPosition.Y);
+                return string.Format(Resources.RectangleRegion_GetColorPickerText, color.R, color.G, color.B, ColorHelpers.ColorToHex(color), CurrentPosition.X, CurrentPosition.Y, ColorHelpers.ColorToCIELab(color));
```
Comment (Jaex) [inline ShareX.ScreenCaptureLib/Forms/RectangleRegion.cs]: I don't like default rectangle info text to show CIE here.

**Comment by Jaex** [comment]
Probably who opened #1332 would prefer to have these values on color picker form and there was no empty space in that form to put these values therefore I was not considered adding it.

Also you didn't even support this color in color picker format.

So currently it is pretty much not exist anywhere.

**Comment by jibcore** [comment]
Ok, i try to fix it.

File: ShareX.ScreenCaptureLib/Forms/RectangleRegion.cs (hunk 3 above)
Comment (jibcore) [inline ShareX.ScreenCaptureLib/Forms/RectangleRegion.cs]: How you would like to show CIE Lab?

Comment (Jaex) [inline ShareX.ScreenCaptureLib/Forms/RectangleRegion.cs]: $cie maybe?

**Comment by jibcore** [comment]
Damn, i add lab nuds on color picker form and it caused a lot of changes. But how i can add my controls on form like a this?
http://i.imgur.com/rU7TPSI.jpg

**Comment by Jaex** [comment]
Are you using high dpi in Windows? Designer with resx support causing issues and changes so many things and breaks some of them for few users :(

**Comment by jibcore** [comment]
>Are you using high dpi in Windows?
Nope.
I do not know how to solve this problem. Any time when i try to add my NUDs it affects other controls :c
Sry.

**Pull request closed without merging**